rust-runtime/aws-smithy-runtime-api/src/config_bag.rs

```diff
@@ -258,6 +292,7 @@ mod test {
         assert!(final_bag.get::<Prop2>().is_some());
         // we unset prop3
         assert!(final_bag.get::<Prop3>().is_none());
+        println!("{:#?}", final_bag);
```

Comment:
Did you want this here?

Reply:
yeah, just verifying how the bag looks for debug purposes